azure/metric-alerts.json

```diff
@@ -0,0 +1,82 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2018-05-01/subscriptionDeploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "metricAlertsArray": {
+      "type": "array"
+    },
+    "actionGroupResourceId": {
+      "type": "string"
+    }
+  },
+  "variables": {
+    "deploymentUrlBase": "https://raw.githubusercontent.com/SkillsFundingAgency/das-platform-building-blocks/DASD-8884-add-alert-rule-building-block/templates/"
+  },
+  "resources": [
+    {
+      "apiVersion": "2020-10-01",
+      "name": "[concat('metric-alerts-', parameters('metricAlertsArray')[copyIndex()].appInsightsName)]",
+      "resourceGroup": "[parameters('metricAlertsArray')[copyIndex()].appInsightsResourceGroup]",
+      "type": "Microsoft.Resources/deployments",
+      "properties": {
+        "mode": "Incremental",
+        "templateLink": {
+          "uri": "[concat(variables('deploymentUrlBase'),'metric-alert.json')]",
+          "contentVersion": "1.0.0.0"
+        },
+        "parameters": {
+          "alertName": {
+            "value": "[replace(concat(parameters('metricAlertsArray')[copyIndex()].appInsightsName, ' - ', parameters('metricAlertsArray')[copyIndex()].metricName), '/', '-')]"
+          },
+          "alertDescription": {
+            "value": "[concat(parameters('metricAlertsArray')[copyIndex()].metricName, ' ', parameters('metricAlertsArray')[copyIndex()].metricOperator, ' ', parameters('metricAlertsArray')[copyIndex()].metricThreshold)]"
+          },
+          "alertSeverity": {
+            "value": 0
+          },
+          "isEnabled": {
+            "value": true
```

Comment:
Do we want it enabled in every environment?

Reply:
Added areMetricAlertsEnabled parameter, a deployment to PP with disabled alerts before a deployment to PRD with enabled alerts sounds good.